index.html

```diff
@@ -21,20 +21,33 @@
             </nav>
     
           <div class="second-item">
+            <img class="logo" src="./images/logo.png" alt=""> 
                  <ul class="second-li">
                     <li><a class="active" href="index.html">home</a></li>
                     <li><a href="about.html">about</a></li>
                     <li><a href="">join</a></li>
                     <li><a href="">sponsor</a></li
                     <li><a href="#">News</a></li>
                 </ul>
-                <li><a href=""> <img class="logo" src="./images/logo.png" alt=""></a> </li>  
+               
             </div>
+        
+            
       
     </header>
     <section class="container contain">
+        
         <div class="mobi-logo">
-            <img class="logo-1" src="./images/mobile_menu 1.png" alt="">
+            <img id="hammbuger" class="logo-1" src="./images/mobile_menu 1.png" alt="">
+            <img class="x-icon" id="x-icon" src="./images/Icon - Cancel.svg" alt="">
+            <img class="lop" src="./images/logo.png" alt="">
```

Comment:
- [ ] Nice work so far for accessibility kindly add all the image att attributes your entire application